definitions/variable/technology/power-plant.yaml

```diff
@@ -125,86 +127,63 @@
+- Maximum Charge|Electricity|{Electricity Storage Type}:
+    description: Maximum charge when storing energy for a {Electricity Storage Type} unit
     unit: GW
-- Maximum Discharge|Electricity|Hydro|Pumped Storage:
-    description: Maximum discharge when consuming energy from the pumped-storage hydro
+- Maximum Discharge|Electricity|{Electricity Storage Type}:
+    description: Maximum discharge when consuming energy from a {Electricity Storage Type} unit
       unit.
     unit: GW
-- Maximum Storage|Electricity|Hydro|Pumped Storage:
-    description: Maximum volume of a pumped-storage hydro unit expressed in energy
-    unit: GWh
-- Minimum Storage|Electricity|Hydro|Pumped Storage:
-    description: Minimum volume of a pumped-storage hydro unit expressed in energy
-    unit: GWh
-- Pumping Efficiency|Electricity|Hydro|Pumped Storage:
-    description: Efficiency of pumping for a pumped-storage hydro unit
-    unit: '%'
-    skip-region-aggregation: true
-- Turbine Efficiency|Electricity|Hydro|Pumped Storage:
-    description: Efficiency of turbining for a pumped-storage hydro unit
-    unit: '%'
-    skip-region-aggregation: true
 - Load Factor|Electricity|{Variable Renewable Power Type}:
-    description: Load factors for onshore wind power plants
+    description: Load factors for {Variable Renewable Power Type} power plants
     unit: '%'
     skip-region-aggregation: true
-- Maximum Energy Charge|Electricity|Energy Storage System:
-    description: This is the upper bound that energy storage system could charge energy
-      at time "t".
-    unit: GWh
-- Maximum Charge|Electricity|Energy Storage System:
-    description: Maximum charge when storing energy the energy storage system unit.
-    unit: GW
-- Roundtrip Efficiency|Electricity|Energy Storage System|{Energy Storage System Type}:
+- Roundtrip Efficiency|Electricity|{Electricity Storage Type}:
     description: Efficiency when charging energy from the grid to the system.
-    unit: 
-- Maximum Discharge|Electricity|Energy Storage System:
-    description: Maximum discharge when consuming energy from the energy storage system
-      unit.
-    unit: GW
-- Maximum Storage|Electricity|Energy Storage System:
-    description: Maximum energy that can be stored by the energy storage systems.
-    unit: GWh
-- Maximum Storage|Electricity|Energy Storage System|{Energy Storage System Type}:
-    description: Maximum energy that can be stored by the Lithium Iron Batteries systems.
-    unit: GWh
-- Minimum Storage|Electricity|Energy Storage System:
-    description: Minimum energy that can be stored by the energy storage system unit.
-    unit: GWh
-- Spillage|Electricity|Energy Storage System:
-    description: Energy spillage that could not be stored by an energy storage system
-      (ESS). Several energy models include spillage variables related to renewable
+    unit:
+- Spillage|Electricity|{Electricity Storage Type}:
```

Comment:
Spillage should be in 'electricity-operation.yaml' as this is not a fixed characteristic of a unit but a result of "operation"

Reply:
Yeah I got it. Thanks. 
I solve it now.